questionary/prompts/checkbox.py

```diff
@@ -179,8 +194,9 @@ def get_prompt_tokens() -> List[Tuple[str, str]]:
                         "class:instruction",
                         "(Use arrow keys to move, "
                         "<space> to select, "
```

Comment:
Are we happy to say that space can't be used in the search?

Reply:
We have to exclude it, since space is used to select an entry from the list. Otherwise we would have to come up with a rather complex solution to allow (de-)selection of an entry when the search is enabled.

Reply:
Makes sense